website/docs/r/network_watcher_flow_log.html.markdown

```diff
@@ -87,6 +87,10 @@ The following arguments are supported:
 
 * `network_security_group_id` - (Required) The ID of the Network Security Group for which to enable flow logs for. Changing this forces a new resource to be created.
 
+~> **NOTE:** `network_security_group_id` is deprecated and will be removed in favour of the property `target_resource_id` in version 4.0 of the AzureRM Provider.
```

Comment:
Standard practice for deprecated properties is to remove them from the documentation entirely to discourage users from using them instead of leaving a note. 
```suggestion
```